bin/opam.ml

```diff
@@ -191,6 +170,34 @@ let submit ~dry_run ~opam_repo local_repo remote_repo pkgs auto_open =
       | Ok ()   -> Ok 0
       | Error _ -> msg ()
 
+let submit ~dry_run ~yes ~opam_repo local_repo remote_repo pkgs auto_open =
+  Config.token ~dry_run () >>= fun token ->
+  List.fold_left (fun acc pkg ->
+      get_pkg_dir pkg
+      >>= fun pkg_dir -> Sos.dir_exists ~dry_run pkg_dir
+      >>= function
+      | true  -> acc
+      | false ->
+          Logs.err (fun m ->
+              m "Package@ %a@ does@ not@ exist. Did@ you@ forget@ \
```

Comment:
I think Fmt takes care of the `@ ` space mapping for you automatically as part of Logs, so you can just use a normal space

Reply:
This change is actually irrelevant to this PR as this line was just moved but I'm happy to remove those as they'll make the whole thing a lot clearer!

Reply:
glad@ to @ hear@ that! @ :) @ :)